README.md

```diff
@@ -1,6 +1,7 @@
 # PostgreSQLCursor for handling large Result Sets
 
 [![Gem Version](https://badge.fury.io/rb/postgresql_cursor.svg)](http://badge.fury.io/rb/postgresql_cursor)
+[![Build Status](https://travis-ci.org/fgarces/postgresql_cursor.svg?branch=master)](https://travis-ci.org/fgarces/postgresql_cursor)
```

Comment:
This should be changed from `fgarces` to `afair` before merge.

Reply:
I've just set up Travis for this repo. https://travis-ci.org/afair/postgresql_cursor